vhost_user_block/src/lib.rs

```diff
@@ -245,8 +248,10 @@ impl BlockEpollHandler {
             match Request::parse(&head, mem) {
                 Ok(request) => {
                     debug!("element is a valid request");
+                    let mut disk_image_locked = self.disk_image.lock().unwrap();
+                    let mut disk_image = disk_image_locked.deref_mut();
```

Comment:
This would inhibit any significant amount of parallelism in vhost_user_blk. It's fine if this is just an initial implementation to be improved upon in future changes, but if that's the case, I think there should be a TODO comment there.

Reply:
Sorry, I just push a new version but missing your comment here, I will add it later.